external-crates/move/crates/move-compiler/src/cfgir/translate.rs

```diff
@@ -613,14 +615,26 @@ fn function_body(
     let b_ = match tb_ {
         HB::Native => GB::Native,
         HB::Defined { locals, body } => {
+            if DEBUG_PRINT {
```

Comment:
I promised @tnowacki I'd remove these before landing anything in main. I was waiting for all the nits and stuff to be done in case I needed to look at any output. I did remove it, though.

In general I find this sort of stuff laying around the code helpful, because it saves time tracking bugs down by just turning on some printing, so maybe we can do something like this in a principled way later.